src/include/btmem.h

```diff
@@ -899,6 +900,7 @@ struct __wt_update {
 #define	WT_UPDATE_MODIFIED	1	/* partial-update modify value */
 #define	WT_UPDATE_RESERVED	2	/* reserved */
 #define	WT_UPDATE_STANDARD	3	/* complete value */
+#define	WT_UPDATE_IMPOSSIBLE	255
```

Comment:
I can see how you ended up with `WT_UPDATE_IMPOSSIBLE`, but I think it's telling us that we need to refactor `__wt_col_modify` and `__wt_row_modify`.  As far as I can see, all we're really doing with this unused type is checking that when we call those functions to apply an existing update list, they don't allocate any updates.

Could you please have a look at restructuring the code so this isn't necessary?  Can the common path allocate updates first, then call a function to apply them (called directly in the places that are passing `WT_UPDATE_IMPOSSIBLE`)?

If there is no way around it, a better name might be `WT_UPDATE_INVALID`, but it still feels wrong to define a value here that can never appear in the struct.